vsock_sample/py/vsock-sample.py

```diff
@@ -48,10 +48,11 @@ def recv_data(self):
         """Receive data from a remote endpoint"""
         while True:
             (from_client, (remote_cid, remote_port)) = self.sock.accept()
-            data = from_client.recv(1024).decode()
-            if not data:
-                break
-            print(data)
+            while True:
+                data = from_client.recv(1024).decode()
+                if not data:
+                    break
+                print(data)
```

Comment:
Print adds line breaks, which makes it more difficult to make sense of the (binary) data. You can change this behavior by specifying `end=''` for [print](https://docs.python.org/3/library/functions.html#print) or by using `write(sys.stdout, data)` instead, to avoid modifying the output.

Reply:
is there no way to send the data from the vsock channel "at once"?